adapter/internal/adapter/adapter.go

```diff
@@ -117,7 +118,11 @@ func runManagementServer(conf *config.Config, server xdsv3.Server, enforcerServe
 
 	lis, err := net.Listen("tcp", fmt.Sprintf(":%d", port))
 	if err != nil {
-		logger.LoggerMgw.Fatal("failed to listen: ", err)
+		logger.LoggerMgw.ErrorC(logging.ErrorDetails{
+			Message:   fmt.Sprintf("Failed to listen: %v", err.Error()),
```

Comment:
Can we include a port number here ? `Failed to listen on port: <portNumber> , error : ....`

Reply:
fixed in https://github.com/wso2/product-microgateway/pull/2661/commits/19c337629499420a856f33b0453fda379f1022bf